server/package.json

```diff
@@ -20,7 +20,7 @@
     "start": "node lib/index.js",
     "dev": "nodemon --exec ts-node src/index.js",
     "start-debug": "REMOTE_DEBUG=true DEBUG_PORT=60101 node --inspect=60101 lib/index.js",
-    "lint": "eslint . --ext .js --quiet",
+    "lint": "eslint src --ext .js --quiet && tslint --project tsconfig.json src/**/*.ts",
```

Comment:
Like this, I think I might transplant some of this stuff over to the EM monitor!